src/mappers/ItemVisitor.ts

```diff
@@ -28,11 +28,16 @@ export class ItemVisitor {
                     text: `£${item.item.totalItemCost}`
                 },
                 {
-                    html: `<a class="govuk-link" href="${this.getViewChangeCertOptionsLink(item.item.id, itemOptions.companyType)}">View/Change certificate options</a>`
+                    html: `<a class="govuk-link" href="${this.getViewChangeCertOptionsLink(item.item.id, itemOptions.companyType)}">
+                                View/Change certificate options
+                                <span class="govuk-visually-hidden">
+                                    ${itemOptions.companyType} for ${item.item.companyNumber}
```

Comment:
Does the screen reader actually read out `“View/Change <Certificate type> for <Company Number>"` as a result of this?

Reply:
Yes it does, I did try it out